addons/website/data/website_contactus_data.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
TODO: Probably also need to move this back inside website_data.xml where it came from
less diff + with other pages where is was
Or at least adapt the comment `<!-- Template for home and contactus -->` since contactus is not there anymore

Reply:
Done